data/locale/en-US.ini

```diff
@@ -526,3 +526,12 @@ Codec.ProRes.Profile.APCN="422 Standard (APCN)"
 Codec.ProRes.Profile.APCH="422 High Quality/HQ (APCH)"
 Codec.ProRes.Profile.AP4H="4444 High Quality/HQ (AP4H)"
 Codec.ProRes.Profile.AP4X="4444 Extreme Quality/XQ (AP4X)"
+
+# Codec: Avid DNxHR
+Codec.DNxHR.Profile="Profile"
+Codec.DNxHR.Profile.dnxhd="DNxHR"
```

Comment:
Text may be incorrect according to: https://ffmpeg.org/doxygen/3.2/profiles_8c_source.html#l00049

Reply:
Yeah, that's a typo. I fixed it. 
Although the DNxHD profile is hidden. I still added the translation for completeness (not that the profile needs any translation as it is called like this in every language).